pkg/istio-agent/sds-agent.go

```diff
@@ -453,4 +455,8 @@ func applyEnvVars() {
 	workloadSdsCacheOptions.InitialBackoffInMilliSec = int64(initialBackoffInMilliSecEnv)
 	// Disable the secret eviction for istio agent.
 	workloadSdsCacheOptions.EvictionDuration = 0
+	if citadel.ProvCert != "" {
+		workloadSdsCacheOptions.AlwaysValidTokenFlag = true
```

Comment:
a comment here would be useful

Reply:
I will address this later

Reply:
Looks like isTokenExpire() function does not really do anything. Should we remove the function instead of adding these configs?